creator-node/src/middlewares.js

```diff
@@ -32,6 +36,45 @@ async function authMiddleware (req, res, next) {
     wallet: cnodeUser.walletPublicKey,
     cnodeUserUUID: cnodeUserUUID
   }
+
+  next()
+}
+
+/** Conditional logic if call is coming from another creator node */
+async function crossCnodeAuth (req, res, next) {
```

Comment:
so this needs to be caalled after authMiddleware?

Reply:
per discussion, this is temporary and will eventually be combined to authMiddleware. going to do that after completing integration though